kyo-combinators/shared/src/main/scala/kyo/AbortCombinators.scala

```diff
@@ -0,0 +1,637 @@
+package kyo
+
+import kyo.debug.Debug
+import kyo.kernel.ArrowEffect
+import scala.annotation.tailrec
+import scala.annotation.targetName
+import scala.util.NotGiven
+
+extension [A, S, E](effect: A < (Abort[E] & S))
+
+    /** Handles the Abort effect and returns its result as a `Result[E, A]`.
+      *
+      * @return
+      *   A computation that produces the result of this computation with the Abort[E] effect handled
+      */
+    def result(
+        using
+        ct: SafeClassTag[E],
+        fl: Flat[A],
+        fr: Frame
+    ): Result[E, A] < S =
+        Abort.run[E](effect)
+
+    /** Handles the Abort effect and returns its result as a `Result.Partial[E, A]`, not handling Panic exceptions.
+      *
+      * @return
+      *   A computation that produces a partial result of this computation with the Abort[E] effect handled
+      */
+    def partialResult(
```

Comment:
if one of the main goals of combinators is to help with auto completion, wouldn't `resultPartial` and `resultPartialOrThrow` work better?

Reply:
yes -- and in fact now that you mention it, I had trouble finding this method recently. I'll update.